docs/user/guides/canonical-urls.rst

```diff
@@ -1,58 +0,0 @@
-How to enable canonical URLs
-============================
-
-In this guide, we introduce relevant settings for enabling canonical URLs in popular documentation frameworks.
-
-If you need to customize the domain from which your documentation project is served,
-please refer to :doc:`/guides/custom-domains`.
-
-Sphinx
-~~~~~~
-
-If you are using :doc:`Sphinx </intro/getting-started-with-sphinx>`,
-Read the Docs will automatically add a default value of the html_baseurl_ setting matching your canonical domain.
-
-If you are using a custom ``html_baseurl`` in your ``conf.py``,
-you have to ensure that the value is correct.
-This can be complex,
-supporting pull request builds (which are published on a separate domain),
-special branches
-or if you are using :term:`subproject` s or :doc:`translations </localization>`.
-We recommend not including a ``html_baseurl`` in your ``conf.py``,
-and letting Read the Docs define it.
```

Comment:
Does our current guide explain how to do this effectively? I don't think we really need to set canonical URLs in PR builds though, since they are disallowed by `robots.txt`.

Reply:
What's the effectively way of doing this? The current guide just mention that on Sphinx you can use `html_baseurl` and on MkDocs the `site_url` configs: https://docs.readthedocs.io/en/stable/canonical-urls.html#how-to-specify-the-canonical-url

Setting these for specific tools should be done in different pages. See my other comment: https://github.com/readthedocs/readthedocs.org/pull/11516#issuecomment-2271938541